templates/sios-datakeeper-master.template

```diff
@@ -898,15 +813,7 @@
                         "Ref": "QSS3BucketName"
                     },
                     "QSS3KeyPrefix": {
-                        "Fn::Join": [
-                          	"/",
-                           [
-                             	{
-                                	"Ref": "QSS3KeyPrefix"
-                             	},
-                              	"submodules/quickstart-microsoft-rdgateway"
-                           ]
-                        ]
+                        "Fn::Sub": "${QSS3KeyPrefix}submodules/quickstart-microsoft-rdgateway"
```

Comment:
Needs a slash at the end:
"Fn::Sub": "${QSS3KeyPrefix}submodules/quickstart-microsoft-rdgateway/"